### PR TITLE
add inital backstage integration

### PR DESCRIPTION
**Pull request by lcouzens**
## Jira Ticket

[COST-5006](https://issues.redhat.com/browse/COST-5006)

## Description

This change will add a mkdocs.yml for backstage itegration

## Testing

N/A

## Release Notes
- [ ] proposed release note

```markdown
* [COST-####](https://issues.redhat.com/browse/COST-####) N/A
```

### CODE REVIEW

**Review by bacciotti — APPROVED** [review]

**Review by bacciotti — APPROVED** [review]

File: koku/mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+docs_dir: docs/
```
Comment (maskarb) [inline koku/mkdocs.yml:1]: why is this file here instead of the root directory?
